### DISCUSSION TITLE
Add column for "File type" in song lists

### DISCUSSION THREAD

**Discussion by GioF71** [jcorporation/myMPD#1224] [Ideas]
Hello, first of all thank you once again for the great work.

In song lists, like the queue for example, I can currently select and display the "Audio format" column. 
However this column does not tell me about the file format, which is available as "File format" on the detail windows.
I might be wrong but I did not find a way to display this information in the table view.
If this is the case, would it be possible to add the functionality?

Thank you!

**Comment by jcorporation** [comment]
Added with [f2b8d2e](https://github.com/jcorporation/myMPD/commit/f2b8d2eb217a7a8655bf6fc4bc393d000be200c6). The tables are displaying only the short extensions and not the long name.

**Reply by GioF71** [reply]
I suppose that this will be enough! Thank you

**Reply by GioF71** [reply]
Hello, would it be possible to add both "Audio format" and "File type" in the Album detail view as well?
Thanks a lot!

**Reply by jcorporation** [reply]
> Hello, would it be possible to add both "Audio format" and "File type" in the Album detail view as well?

An album has no file type and no audio format, they exists only for songs. You can enable the file type for the song listing in the album view, unfortunately MPD does not provide an audio format output for a search, only for song details and in the queue output.

**Reply by GioF71** [reply]
Hello, yes I was referring to the songs listing in the album view, not to the list of albums. I expected as you say that this cannot be shown at the album level.
But, in a album song listing, I cannot find the "File type" among the columns that can be enabled. Am I missing something?
If necessary I can provide a screenshot or two

**Reply by jcorporation** [reply]
This is added with above commit, it is in the v15.0.0 branch not in current master or devel.

**Reply by GioF71** [reply]
Ok, thank you again!
